lms/static/js/discussions_management/views/discussions.js

```diff
@@ -16,11 +28,148 @@
                 },
 
                 render: function() {
-                    HtmlUtils.setHtml(this.$el, this.template({}));
+                    var selectedScheme, topicNav;
+                    HtmlUtils.setHtml(this.$el, this.template({
+                        availableSchemes: this.getDivisionSchemeData(this.discussionSettings.attributes.division_scheme)
+                    }));
+                    selectedScheme = this.getSelectedScheme();
+                    topicNav = this.getTopicNav();
+                    this.hideTopicNav(selectedScheme, topicNav);
                     this.showDiscussionTopics();
                     return this;
                 },
 
+                getDivisionSchemeData: function(selectedScheme) {
+                    var self = this;
+                    return [
+                        {
+                            key: none,
+                            displayName: gettext('Not divided'),
+                            descriptiveText: gettext('Discussions are unified; all learners interact with posts from other learners, regardless of the group they are in.'), //  eslint-disable-line max-len
+                            selected: selectedScheme === none,
+                            enabled: true // always leave none enabled
+                        },
+                        {
+                            key: enrollmentTrack,
+                            displayName: gettext('Enrollment Tracks'),
+                            descriptiveText: gettext('Use enrollment tracks as the basis for dividing discussions. All learners, regardless of their enrollment track, see the same discussion topics, but within divided topics, only learners who are in the same enrollment track see and respond to each others’ posts.'), //  eslint-disable-line max-len
+                            selected: selectedScheme === enrollmentTrack,
+                            enabled: self.isSchemeAvailable(enrollmentTrack) || selectedScheme === enrollmentTrack
+                        },
+                        {
+                            key: cohort,
+                            displayName: gettext('Cohorts'),
+                            descriptiveText: gettext('Use cohorts as the basis for dividing discussions. All learners, regardless of cohort, see the same discussion topics, but within divided topics, only members of the same cohort see and respond to each others’ posts. '), //  eslint-disable-line max-len
+                            selected: selectedScheme === cohort,
+                            enabled: self.isSchemeAvailable(cohort) || selectedScheme === cohort
+                        }
+
+                    ];
+                },
+
+                isSchemeAvailable: function(scheme) {
+                    var self = this;
```

Comment:
No need to declare self here. I think this might be a common pattern in your code.